app/components/publications/batch-documents-publication-modal.hbs

```diff
@@ -0,0 +1,51 @@
+<WebComponents::AuModal>
+  <WebComponents::AuModal::Header @title={{ t "documents" }} @closeModal={{this.onCancel}} />
+  <WebComponents::AuModal::Body>
+    {{!-- template-lint-disable table-groups --}}
+    <table class="auk-table auk-table--styled auk-table--bordered">
+      <thead>
+        <tr>
+          <th>{{t "name"}}</th>
+          <th>{{t "document-type"}}</th>
+          <th></th>
+        </tr>
+      </thead>
+
+      <Documents::DocumentList
+        @pieces={{this.pieces}}
+        {{!-- should be tbody tag; workaround for Documents::DocumentList rendering a div --}}
+        style="display: table-row-group;"
+        as |pieces|>
+        {{#each pieces as |piece|}}
+          <tr>
+            <td>
+              {{piece.name}}
+            </td>
+            <td>
+              {{piece.file.extension}}
+            </td>
+            <td>
+              <WebComponents::AuButtonToolbar>
+                <WebComponents::AuButton
+                  @layout="icon-only"
+                  @icon="download"
+                  @skin="borderless"
+                  @href="{{await piece.downloadFileLink}}"
+                  download
```

Comment:
Wat is de betekenis hiervan?

Reply:
Dit geeft aan dat de browser moet downloaden, niet surfen naar.

Reply:
Voor het meegeven van de naam van de download, zou `download` in theorie van pas kunnen komen, maar omwille van de content-disposition-header wordt deze toch teniet gedaan. Zie https://github.com/kanselarij-vlaanderen/kaleidos-frontend/blob/development/app/components/web-components/vl-document.hbs#L41 ... hij betekent hier dus geen meerwaarde denk ik

Reply:
Nee, als ik het weglaat, download hij niet, maar surft hij naar de link die niet bestaat. De content-disposition-header override enkel de filenaam is, vandaar dat ik deze heb weggelaten (als `download="file1.pdf"`)